packages/component-utils/lib/core/setup.ts

```diff
@@ -0,0 +1,36 @@
+import { ComponentTestInstance, MountOptions } from './options'
+import { handleError } from './errors'
+import { getRoot, getDocument } from './util/dom'
+import { injectStylesBeforeElement } from './util/styles'
+
+function checkMountModeEnabled () {
+  // @ts-ignore
+  if (Cypress.spec.specType !== 'component') {
+    throw new Error(
+      `In order to use mount or unmount functions please place the spec in the component folder defined in cypress.json`,
```

Comment:
I don't see where the unmount function is defined, does that come from the individual component testing plugin?

Also, it might be clearer to throw this error with the user's actual `componentFolder` path (or whatever the config key is)

Reply:
It used to exist and will be defined in the framework adapter until I remove it.

It would definitely be nice of us to do that. Not doing that right now.

Reply:
Ah, I see an example of it being used now: https://github.com/bahmutov/cypress-react-unit-test/blob/master/cypress/component/basic/unmount/comp-spec.js